libexec/bats-core/bats-exec-file

```diff
@@ -49,6 +49,12 @@ fi
 
 BATS_TEST_FILENAME="$filename"
 
+if ! BATS_FILE_TMPDIR=$(mktemp -d "${BATS_SUITE_TMPDIR}/bats-file-tmpdir-XXXXXX");then
```

Comment:
I think it would be easier for debugging/navigation to have the filename as part of the tmpdir. Maybe `${BATS_SUITE_TMPDIR}/file-tmpdir-$(basename "${BATS_FILENAME}")-XXXX`.